strings/phone_validator.py

```diff
@@ -0,0 +1,30 @@
+import re
+def phone_validator(phone: str) -> bool:
+    """
+    Determine whether the string is a valid phone number or not
+    :param phone:
+    :return: Boolean
+    >>> phone_validator("+91123456789")
+    False
+    >>> phone_validator("+919876543210")
+    True
+    >>> phone_validator("01234567896")
+    False
+    >>> phone_validator("919876543218")
+    True
+    >>> phone_validator("+91-1234567899")
+    False
+    """
+    # Created a regex pattern to produce a valid phone number
+    pat = re.compile(r"^(\+91[\-\s]?)?[0]?(91)?[789]\d{9}$")
```

Comment:
Why only +91 ? It's not very universal. Consider create function for each country.

Reply:
Ok, I change this function as a indian_phone_validator.